src/components/character-count/template.njk

```diff
@@ -0,0 +1,36 @@
+{% from "../error-message/macro.njk" import govukErrorMessage -%}
+{% from "../hint/macro.njk" import govukHint %}
+{% from "../label/macro.njk" import govukLabel %}
+{% from "../textarea/macro.njk" import govukTextarea %}
+
+{#- a record of other elements that we need to associate with the input using
+   aria-describedby – for example hints or error messages -#}
+{% set describedBy = "" %}
```

Comment:
I'm not following how describedBy gets set to something other than '', where does that happen?

Reply:
I added the id of the default message as a default value for describedBy - which can be amended with hint and error message ids in the textarea component.